### PR TITLE
Update bookings-in-outlook.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about EwsApplicationAccessPolicy parameter that it defines which applications other than Entourage, Mac Outlook, and Outlook can access EWS.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8625

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -107,6 +107,10 @@ Use the **Get-OrganizationConfig** and **Set-OrganizationConfig** commands to fi
    ```PowerShell
    Set-OrganizationConfig -EwsApplicationAccessPolicy EnforceBlockList -EwsBlockList @{Add="MicrosoftOWSPersonalBookings"}
    ```
+
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md:110]:
```suggestion
 ```

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 2]
```diff
@@ -107,6 +107,10 @@ Use the **Get-OrganizationConfig** and **Set-OrganizationConfig** commands to fi
    ```PowerShell
    Set-OrganizationConfig -EwsApplicationAccessPolicy EnforceBlockList -EwsBlockList @{Add="MicrosoftOWSPersonalBookings"}
    ```
+   
+   
+  > [!NOTE]
+  > The EwsApplicationAccessPolicy parameter defines which applications other than Entourage, Mac Outlook, and Outlook can access EWS.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
  > The EwsApplicationAccessPolicy parameter defines which applications other than Entourage, Outlook, and Outlook for Mac can access EWS.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**